python/setup.py

```diff
@@ -494,6 +493,7 @@ def parse_version(root):
     tests_require=['pytest', 'pandas'],
     description="Python library for Apache Arrow",
     long_description=long_description,
+    long_description_content_type="text/markdown",
```

Comment:
Are unknown arguments simply ignored? I'm thinking of people who may want to build Arrow using an older setuptools.

Reply:
I can create a new environment with `setuptools < 38.6.0` to see if building breaks.

Reply:
I get this warning: `UserWarning: Unknown distribution options: 'long_description_content_type' warnings.warn(msg)` but the build still completes when running `python setup.py build_ext --build-type=$ARROW_BUILD_TYPE --with-parquet --with-plasma --inplace` do you have any suggestions on a more formal or exhaustive way to check on this?

Reply:
No, it's fine with a warning, as long as the build succeeds.